core/src/main/java/org/infinispan/factories/KnownComponentNames.java

```diff
@@ -31,6 +31,7 @@ public class KnownComponentNames {
    public static final String INTERNAL_MARSHALLER = "org.infinispan.marshaller.internal";
    public static final String PERSISTENCE_MARSHALLER = "org.infinispan.marshaller.persistence";
    public static final String USER_MARSHALLER = "org.infinispan.marshaller.user";
+   public static final String TELEMETRY_SERVICE = "org.infinispan.telemetry.service";
```

Comment:
this is not necessary. Component names are only required if you need multiple implementations of the same interface. As far as I can tell, you only have `OpenTelemetryService` as implementation.